robots/bluevolta_description/urdf/bluevolta_bravo7_sphere_endeffector.urdf

```diff
@@ -0,0 +1,403 @@
+<?xml version="1.0"?>
+<robot name="bluevolta">
```

Comment:
1. Rename this URDF as `bluevolta_bravo7_no_ee.urdf`
2. Adapt the URDF as follows
```suggestion
<robot name="bluevolta_bravo7_no_ee">
```